Assets/Scripts/PlayerController.cs

```diff
@@ -65,7 +65,8 @@ void UpdateAnimationAndMove()
         if (change != Vector3.zero)
         {
             MoveCharacter();
-            
+            change.x = Mathf.Round(change.x);
```

Comment:
What is the purpose behind rounding?

Reply:
Added 4 more motion fields therefore any value that change.x or change.y gets, it ends up in one of the eight places.